.cirrus.yml

```diff
@@ -0,0 +1,37 @@
+# By default, Cirrus mounts an empty volume to `/tmp`
+# which triggers all sorts of warnings like "system temporary path is world-writable: /tmp".
+# Lets workaround it by specifying a custom volume mount point.
+env:
+  CIRRUS_VOLUME: /cirrus
+
+task:
+  name: Test on Arm64 Graviton2 ($CC)
+  arm_container:
+    image: ghcr.io/ruby/ruby-ci-image:$CC
+    cpu: 8
+    memory: 32G
```

Comment:
I am working on my branch on my fork based on the commit on this PR.
I am looking for the reason of `memory: 32G`. And I found the following article.

https://cirrus-ci.org/guide/linux/
> Containers on Community Cluster can use maximum 8.0 CPUs and up to 32 GB of memory. 

So, the maximum memory of each container case in the matrix is 32 GB. That's why you did set the `32G`, right? Is it a total value of the entire matrix cases like `cpu: 8`? If the value is shared in the current 2 matrix cases, do we have to set `16G` instead of `32G` on this line?

By the way, I heard from you that the maximum cpu is 16 for Arm, not 8. So, the maximum 8 CPUs is for only x86_64 specific?

Reply:
In this scenario each of the tasks are getting 8 CPUs and `32G` of memory. The rule is that you can request `4G` of memory for each core. For example, for a tasks with `0.25` CPU you only can request up to `1G`.

16 CPUs is the total amount of CPUs a user can use in parallel before tasks being queued. With Cirrus you have per-user limits rather then per-organization like with GHA. You can split these 16 CPUs in any amount of parallel tasks.

Reply:
Okay. So, when we can use 8 cores for 1 job, we can request maximum 32 GB (= 8 cpu x 4 GB). I assume that that's the reason the 32G is set on the line.

Reply:
Yes, exactly. 👌